examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -380,9 +380,9 @@ func (n *dcFnNamer) Name(t *types.Type) string {
 	pubName := n.public.Name(t)
 	n.tracker.AddType(t)
 	if t.Name.Package == n.myPackage {
-		return "DeepCopy_" + pubName
+		return "DeepCopy" + pubName
 	}
-	return fmt.Sprintf("%s.DeepCopy_%s", n.tracker.LocalNameOf(t.Name.Package), pubName)
+	return fmt.Sprintf("%s.DeepCopy%s", n.tracker.LocalNameOf(t.Name.Package), pubName)
```

Comment:
@duglin @smarterclayton as you might already noticed, the lint errors clutters the test output and as @duglin said unless we skip it, the cluttered test output can be a painful reading.